java/meterpreter/meterpreter/src/main/java/com/metasploit/meterpreter/TcpTransport.java

```diff
@@ -49,7 +49,7 @@ public void run() {
                     try {
                         this.in.close();
                     }
-                    catch (IOException ex2) {
+                    catch (IOException ignored) {
```

Comment:
Was the linter okay with `ex` normally?

Reply:
Oh, looks like there's other lines changing `ex` to `ignored` - so I'm not sure now.